xml/System.Runtime.InteropServices/UnmanagedCallersOnlyAttribute.xml

```diff
@@ -132,7 +132,15 @@ Supplied types must be from the official <xref:System.Runtime.CompilerServices>
       </ReturnValue>
       <Docs>
         <summary>Optional. If omitted, no named export is emitted during compilation.</summary>
-        <remarks>To be added.</remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The export name must be a valid identifier for a native symbol on the target platform. The identifier must not conflict with system provided native identifiers (such as: names of methods in the C standard library, or POSIX/Win32 API names).
+
+Target-dependent name mangling will be performed on the symbol name such as prepending/appending `_` or `@` characters to distinguish calling conventions on x86 Windows.
```

Comment:
```suggestion
During AOT compilation, target-dependent name mangling will be performed on the symbol name such as prepending/appending `_` or `@` characters to distinguish calling conventions on x86 Windows.
```

This is also limited to Native AOT compilation. I assume this is also true for llvm-aot as well?

/cc @lambdageek @vargaz

Reply:
Yes, mono adds a '_' prefix as well if needed.

Reply:
Here's what I'm observing with DNNE:

User is asking to export FancyName here:

https://github.com/AaronRobinsonMSFT/DNNE/blob/3e39b7ca22dea40c4346be7e94810248ab402b38/sample/Exports.cs#L26-L28

I `dotnet build --runtime win-x86 Sample.csproj`

`dumpbin /exports` says `_FancyName@4` got exported.

This is documenting that. I'm doing this for DNNE's sake, NativeAOT doesn't work on x86 Windows. But this feels like the expected behavior.

Reply:
Shoot. I'm confusing this with the other scenario involving stdcall and expecting cdecl. I assume NAOT is also just deferring to the linker in this case, the same DNNE. DNNE can override the default symbol generation but only on Windows and only through a `.def` file.

Reply:
> I assume NAOT is also just deferring to the linker in this case

Linker doesn't know the calling convention or signature of the symbol so it can't do it. NAOT will have to do this mangling manually like the C compiler does. We don't currently have that code because of no x86 Windows support to begin with. We do prepend `_` on Apple platforms though.